plugins/commands/requirements/src/main/kotlin/RequirementsCommand.kt

```diff
@@ -177,20 +182,20 @@ class RequirementsCommand : OrtCommand(
         }
 
         echo("Prefix legend:")
-        echo("\t- The tool was not found in the PATH environment.")
-        echo("\t+ The tool was found in the PATH environment, but not in the required version.")
-        echo("\t* The tool was found in the PATH environment in the required version.")
+        echo("${DANGER_PREFIX}The tool was not found in the PATH environment.")
```

Comment:
Would it make sense to remove the trailing whitespace from the prefixes and get rid of the curly braces?

Reply:
I was a bit undecided about this myself, but in end went for the current solution to also reduce the duplication of the trailing whitespaces, and the `..._PREFIX` variable names IMO fit better if the whitespaces are included.